src/Framework/PaymentGateways/DonationSummary.php

```diff
@@ -0,0 +1,120 @@
+<?php
+
+namespace Give\Framework\PaymentGateways;
+
+use Give\PaymentGateways\DataTransferObjects\GatewayPaymentData;
+
+/**
+ * @unreleased
+ */
+class DonationSummary
```

Comment:
@kjohnson which function in this class return value equivalent to `give_payment_gateway_item_title`?

Reply:
I'm not seeing `give_payment_gateway_item_title()` used by `give_payment_gateway_donation_summary()`. Instead, the original function checks the donation data for a passed form title.

```php
// function give_payment_gateway_donation_summary

// Form title.
$summary = ( ! empty( $donation_data['post_data']['give-form-title'] )
  ? $donation_data['post_data']['give-form-title']
  : ( ! empty( $form_id )
	  ? wp_sprintf( __( 'Donation Form ID: %d', 'give' ), $form_id )
	  : __( 'Untitled donation form', 'give' ) ) );
```